RecoHGCal/TICL/python/EMStep_cff.py

```diff
@@ -29,6 +29,7 @@
         shower_start_max_layer = 5, #inclusive
         max_out_in_hops = 1,
         skip_layers = 2,
+        siblings_maxRSquared = [6e-4, 6e-4, 6e-4],
```

Comment:
```suggestion
```
not needed here (the same as the default already).

Same comment for the other files below